Dockerfile

```diff
@@ -1,6 +1,6 @@
 ### Standard binary
 # Build the manager binary
-FROM golang:1.21.6 as builder
+FROM golang:1.21 as builder
```

Comment:
Through this modification, it is indeed possible to use the latest version of the image when building an image. However, we cannot determine which specific version it is. 

I tend to still use the specified version of the image.

Reply:
💯 reproducible builds. I agree with @tao12345666333